packages/cli/src/lib/flags/snapshotCommonFlags.ts

```diff
@@ -21,3 +26,14 @@ export const sync = () => ({
       'Apply synchronization when there is a 100% match between organization and snapshot resources.',
   }),
 });
+
+export const previewLevel = () => ({
+  previewLevel: flags.enum({
+    char: 'p',
+    description:
+      'The verbosity of the preview. The `light` preview is faster to generate but only contains a limited amount of information, as opposed to the `detailed` preview that takes more time to generate, but returns a diff representation of all the changes to apply.',
+    options: [PreviewLevel.Light, PreviewLevel.Detailed],
```

Comment:
```suggestion
    options: Object.keys(PreviewLevelValue),
```

Like this, if for whatever reason we add a new PreviewLevel, we can keep this value 🎉 .

Reply:
Good point. but I think you meant `Object.values`.